5calls/app/src/main/java/org/a5calls/android/a5calls/controller/IssueActivity.java

```diff
@@ -251,7 +253,8 @@ public void onBackPressed() {
     private void reportCall(String callType, String zip) {
         setButtonsEnabled(false);
         AppSingleton.getInstance(getApplicationContext()).getDatabaseHelper().addCall(mIssue.id,
-                mIssue.contacts[mActiveContactIndex].id, callType, zip);
+                mIssue.name, mIssue.contacts[mActiveContactIndex].id,
```

Comment:
Oh interesting, this is how we do it on my team at work -- just wrap after the comma or + etc to hit 100 line lengths. I will probably keep doing this. https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping has no particular guidelines either way.